src/main/java/com/pokegoapi/auth/GoogleLoginSecrets.java

```diff
@@ -0,0 +1,14 @@
+package com.pokegoapi.auth;
+
+/**
+ * Created by RajulB on 7/23/2016.
+ */
+
+public class GoogleLoginSecrets {
+	public static final String SECRET = "NCjF1TLi2CcY6t5mt0ZveuL7";
+	public static final String CLIENT_ID = "848232511240-73ri3t7plvk96pj4f85uj8otdat2alem.apps.googleusercontent.com";
+	public static final String OAUTH_ENDPOINT = "https://accounts.google.com/o/oauth2/device/code";
+	public static final String OAUTH_TOKEN_ENDPOINT = "https://www.googleapis.com/oauth2/v4/token";
+
+	public static String refresh_token = null;
```

Comment:
You should add documentation to this. As soon as the application is closed this memory is cleared and the token is gone. Doing it this was requires the application to set this static field before logging back in, assuming they also acquired the initial access token.